docs/contribute.md

```diff
@@ -124,3 +124,32 @@ For a specific task
 ```bash
 node make.js testLegacy --task Xcode
 ```
+
+# Package
+
+After a local code update and testing, you should test any changes in your Azure DevOps organization before submitting a pull request. 
+
+## Prepare your organization
+In order to upload an extension directly to your Azure DevOps organization you must be an [administrator](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=vsts#security) of "All agent pools" or have a PAT token associated with an administrator.  
```

Comment:
Maybe add a recommendation along the lines of "we recommend using a dedicated organization for tests like this"